src/app/components/PortraitVideoCarousel/PortraitVideoCarouselNavigation/index.tsx

```diff
@@ -8,28 +8,31 @@ import {
 import { ServiceContext } from '#app/contexts/ServiceContext';
 import styles from './index.styles';
 import { LeftChevron, RightChevron } from '../../icons';
-import { PROMO_ITEM_WIDTH_MIN } from '../constants';
+import { PROMO_ITEM_WIDTH_MIN } from '../styleUtils';
 
-const defaultTranslations = {
-  scrollLeft: 'Scroll left',
-  scrollRight: 'Scroll right',
+const DEFAULT_TRANSLATION = {
+  scrollToPrevious: 'Scroll to previous item',
+  scrollToNext: 'Scroll to next item',
 };
 
 export default ({ scrollPaneRef }: PortraitVideoCarouselNavigationProps) => {
   const {
     dir,
-    translations: { carousel = defaultTranslations },
+    translations: { carousel = DEFAULT_TRANSLATION },
   } = useContext(ServiceContext);
   const [canScrollLeft, setCanScrollLeft] = useState(false);
   const [canScrollRight, setCanScrollRight] = useState(false);
 
-  const { scrollLeft: scrollLeftAria, scrollRight: scrollRightAria } = carousel;
+  const { scrollToPrevious: scrollLeftAria, scrollToNext: scrollRightAria } =
```

Comment:
Should we also consider removing left & right wording from these variables, and use previous & next instead?